konfera/views.py

```diff
@@ -31,6 +32,10 @@ def event_speakers_list_view(request, event_slug):
 
 def event_list(request):
     events = Event.objects.all().order_by('date_from')
```

Comment:
I know you haven't write this, but please change this to filter only published events...